marxs/optics/multiLayerMirror.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+from astropy.io import ascii
+
+from .base import FlatOpticalElement
+from ..math.pluecker import *
+
+
+class MultiLayerMirror(FlatOpticalElement):
+    '''Multilayer mirror with varying layer thickness along one axis
+    
+    The distance between layers changes along the local y axis.
+    All reflectivity data is assumed to be for a single, desired angle. There
+    is currently no way to enter varying reflection that depends on the angle
+    of incidence.
+    Provide reflectivity data in a file with columns:
+    	'X(mm)' - position along the "changing" axis
+    	'Peak lambda' - wavelength with maximum reflection at a given position
+    	'Peak' - maximum reflection at a given position
+    	'FWHM(nm)' - full width half max, measure of width of reflection Gaussian peaks
+    
+    Parameters
+    ----------
+    fileCode: string
+    	serial number of mirror, used to open reflection data file
+    '''
+    def __init__(self, fileCode, **kwargs):
+        self.fileName = './marxs/optics/docs/' + fileCode + '.txt'
+        if ('zoom' not in kwargs):
+        	kwargs['zoom'] = np.array([1, 24.5, 12])   # in mm
```

Comment:
And if `'zoom' in kwargs`? Would this work with an arbitrary zoom? (That probably depends on the data file loaded? I think Herman has mirrors of different sizes). Is the size somewhere in the datafile? Could that be used to check it's consistent?